cmd/admin-handlers.go

```diff
@@ -1768,6 +1769,36 @@ func getServerInfo(ctx context.Context, r *http.Request) madmin.InfoMessage {
 	}
 }
 
+func getKubernetesInfo() madmin.KubernetesInfo {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	ki := madmin.KubernetesInfo{}
+
+	req, e := http.NewRequestWithContext(ctx, http.MethodGet, kubernetesVersionEndpoint, nil)
```

Comment:
e as error is only used in mc not in server you can use err

Reply:
Done